docs/ocis/adr/0023-index-and-store-metadata.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "23. Index and store metadata"
+date: 2023-10-17T15:15:00+01:00
+weight: 23
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/adr
+geekdocFilePath: 0023-index-and-store-metadata.md
+---
+
+
+* Status: proposed
+* Deciders: @butonic, @micbar
+* Date: 2023-10-17
+
+## Context and Problem Statement
+
+ownCloud Infinite Scale is supposed to become a data platform and as such it needs to provide access to metadata.
+Currently only metadata common to all file types (filesize, mime-type, ...) is stored in the index and the metadata storage.
+We want to make other file type specific metadata available to consumers of our internal and external APIs.
+Simple examples would be audio metadata like artist, album and title or exif metadata in images.
```

Comment:
```suggestion
Metadata of audio- and image data should be made available to consumers of our internal and external APIs.
For both types the Graph API 1.0 has pre defined facets.

This ADR is not talking about a generic metadata storing and retrieval system.
```

Reply:
I think I still disagree, this is putting too much assumptions into the context and problem statement - the context is that we want to make metadata available, the outcome is that we want to focus on graph api because it's simple and reasonable to implement.

This ADR *is* talking about generic metadata storage/retrieval, it explains why we decided against that for now

Reply:
see `### Store everything from extractors`

Reply:
i agree with you @dschmidt